test/utils.js

```diff
@@ -233,3 +234,22 @@ function statsUrl () {
   const future = futureISO.split('T')[0]
   return `/v2/wallet/stats/${date}/${future}`
 }
+
+function makeSettlement (type, balance, overwrites = {}) {
+  const amount = new BigNumber(balance).times(1e18)
+  const fees = amount.times(0.05)
+  const probi = amount.times(0.95)
+  return Object.assign({
+    type,
+    currency: 'USD',
+    altcurrency: 'BAT',
+    fees: fees.toString(),
+    probi: probi.toString(),
+    amount: amount.dividedBy(1e18).toString(),
+    publisher: braveYoutubePublisher,
+    owner: braveYoutubeOwner,
+    transactionId: uuid.v4(),
+    address: uuid.v4(),
+    hash: uuid.v4()
+  }, overwrites)
+}
```

Comment:
Is this function being called?

Reply:
looks like it was not. was meant to be an abstraction. just used